SPECS/kernel/config

```diff
@@ -737,6 +737,7 @@ CONFIG_AS_SHA256_NI=y
 CONFIG_AS_TPAUSE=y
 CONFIG_AS_GFNI=y
 CONFIG_AS_WRUSS=y
+CONFIG_ARCH_CONFIGURES_CPU_MITIGATIONS=y
```

Comment:
result of commit: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/8292f4f8dd1b005d0688d726261004f816ef730a